regression-test/suites/fault_injection_p0/test_load_stream_fault_injection.groovy

```diff
@@ -161,14 +161,10 @@ suite("load_stream_fault_injection", "nonConcurrent") {
     load_with_injection("LoadStream._dispatch.unknown_srcid", "")
 
     // LoadStream meets StreamRPC idle timeout
-    get_be_param("load_stream_idle_timeout_ms")
-    set_be_param("load_stream_idle_timeout_ms", 500)
     try {
         load_with_injection("LoadStreamStub._send_with_retry.delay_before_send", "")
```

Comment:
enable both delay_before_send and short_idle_timeout before actual load to trigger on_idle_timeout?